homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -67,6 +68,7 @@ async def async_setup_entry(
     )
 
     await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
+    async_setup_services(hass, config_entry)
```

Comment:
services should be created in `async_setup` and be independent of the config entry